Sources/StreamChatSwiftUI/ChatChannel/ChannelHeader/ChatChannelHeaderViewModifier.swift

```diff
@@ -30,12 +30,20 @@ public struct DefaultChatChannelHeader: ToolbarContent {
     
     public var body: some ToolbarContent {
         ToolbarItem(placement: .principal) {
-            VStack {
+            VStack(spacing: 2) {
                 Text(channelNamer(channel, currentUserId) ?? "")
                     .font(fonts.bodyBold)
-                Text(channel.onlineInfoText(currentUserId: currentUserId))
-                    .font(fonts.footnote)
-                    .foregroundColor(Color(colors.textLowEmphasis))
+                if !channel.currentlyTypingUsersFiltered(currentUserId: currentUserId).isEmpty
+                    && utils.typingIndicatorPlacement == .navigationBar {
+                    HStack {
+                        TypingIndicatorView()
+                        SubtitleText(text: channel.typingIndicatorString(currentUserId: currentUserId))
+                    }
```

Comment:
Pretty random thought: One approach to being able to work with previews more, could be to refactor stuff like this into its own views. That would drive up the amount of views we have but at the same time would allow for more preview-based UI development (which I'm a big fan of).
So, no feedback here, just a thought.

Reply:
Yes, we can easily do that with the stateless views. We can also think about having separate folder with previews, to keep them separate from the source files.